styles.css

```diff
@@ -0,0 +1,5 @@
+.favourite{
+  font-size:56px;
+  color: red;
+  
+}
```

Comment:
- [ ] You should give style to your h1 tag. Please check the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).).